test/build.spec.js

```diff
@@ -0,0 +1,11 @@
+const Maker = require('../dist/src');
```

Comment:
does it work if you just require from `'../dist'`? that would be ideal if it does -- it would be indicating that `package.json` has the correct value for `main`

Reply:
Good point, yeah it does work with just `../dist`